include/linkdefs.h

```diff
@@ -13,8 +13,8 @@
 #include <stdint.h>
 
 #define RGBDS_OBJECT_VERSION_STRING "RGB%1hhu"
-#define RGBDS_OBJECT_VERSION_NUMBER (uint8_t)9
-#define RGBDS_OBJECT_REV 4
+#define RGBDS_OBJECT_VERSION_NUMBER UINT8_C(9)
+#define RGBDS_OBJECT_REV UINT32_C(4)
```

Comment:
`UINT8_C(9)` will invariably resolve to either `9` or `9U`, since there's no suffixes for types narrower than `int` (nor they would be at all useful, given integer promotion rules). In general there are very few good reasons to use these type conversions at all, and they'd probably be better off as simple constants.

Reply:
What about `sizeof` though?

Reply:
`sizeof` on a constant is generally a good sign that something's wrong.

Reply:
@ISSOtm It's not a problem due to promotion, I'll fix it.